server/admin_management/api/app/routers/router_guards.py

```diff
@@ -121,71 +115,134 @@ async def get_target_user_from_id(
     access_control_privilege_service: AccessControlPrivilegeService = Depends(
         access_control_privilege_service_instance
     ),
-) -> Union[TargetUser, None]:
+) -> TargetUser:
```

Comment:
Always return TargetUser now. It can be existing user (in db) or new user (from request body)